ledger/ledger-on-memory/src/test/suite/scala/com/daml/ledger/on/memory/InMemoryBatchedLedgerReaderWriterIntegrationSpec.scala

```diff
@@ -26,7 +24,7 @@ class InMemoryBatchedLedgerReaderWriterIntegrationSpec(enableBatching: Boolean =
   private val batchingLedgerWriterConfig =
     BatchingLedgerWriterConfig(
       enableBatching = enableBatching,
-      maxBatchQueueSize = 100,
+      maxBatchQueueSize = 1000,
```

Comment:
Why is it needed?

Reply:
I have been trying to fix the tests, this change is not required.